samples/getting_started/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import {v2 as cloudinary} from 'cloudinary';
+
+// Update to your credentials
+cloudinary.config({
+    cloud_name: '<cloud_name>',
+    api_key: '<api_key>',
+    api_secret: '<api_secret>'
+});
+
+// Ensure that you're registered to auto tagging https://cloudinary.com/documentation/google_auto_tagging_addon
+// Ensure that you're registered to background removal https://cloudinary.com/documentation/cloudinary_ai_background_removal_addon
+//Upload an image with auto tag
+//apply eager transformation
+cloudinary.uploader.upload(
+    "https://res.cloudinary.com/demo/image/upload/v1707306308/cld-docs-hp/walking_woman",
+    {
+        public_id: "walking_woman",
+        categorization: "google_tagging",
+        eager: [{effect: "background_removal"}]
+    })
+    .then((result)=>{
+        console.log(result.info.categorization.google_tagging)}).catch((error)=> {console.log(error)});
+
+// log in to you cloudinary account. Navigate to the newly uploaded `walking_woman` image. Select open on the image
+// and navigate to the Metadata tab. Select Auto tags, select plus on the Overcoat tag to tag your image.
+
+//Upload an image for you underlay
+cloudinary.uploader.upload(
+    "https://res.cloudinary.com/demo/image/upload/v17073124415555/cld-docs-hp/street.jpg",
+    { public_id: "street" })
+    .then((result)=>{
+        console.log(result)}).catch((error)=> {console.log(error)});
+
+
+//Transformation image
+const url = cloudinary.url("walking_woman", {
+    transformation: [
+        { effect: "background_removal" },
+        { gravity: "auto", crop: "auto", width: 450 },
```

Comment:
It doesn't make sense to do g_auto and c_auto, when only changing the width, since that will just scale down the whole image and not actually do any smart cropping.  In my discussion today with Keren, we discussed changing it to a square using `ar_1.0` along with the width param.